jenkins/pipelines/cd/atom-jobs/upload-pipeline-run-data-to-db.groovy

```diff
@@ -0,0 +1,99 @@
+package cd
+
+properties([
+        parameters([
+                string(
+                        defaultValue: '',
+                        name: 'PIPELINE_NAME',
+                        description: '',
+                        trim: true
+                ),
+                string(
+                        defaultValue: '',
+                        name: 'PIPELINE_TYPE',
+                        description: '',
+                        trim: true
+                ),
+                string(
+                        defaultValue: '',
+                        name: 'STATUS',
+                        description: '',
+                        trim: true
+                ),
+                string(
+                        defaultValue: '',
+                        name: 'JENKINS_BUILD_ID',
+                        description: '',
+                        trim: true
+                ),
+                string(
+                        defaultValue: '',
+                        name: 'JENKINS_RUN_URL',
+                        description: '',
+                        trim: true
+                ),
+                string(
+                        defaultValue: '',
+                        name: 'PIPELINE_REVOKER',
+                        description: '',
+                        trim: true
+                ),
```

Comment:
Please do not define params in pipeline, migrate them to job DSL, and use simple one line style for multi params.

Reply:
one-line style can not set trim